docs/operations/licensing.md

```diff
@@ -137,26 +154,91 @@ Created at:       2020-03-25T13:48:33Z (1 hour ago)
 Updated at:       2020-03-25T13:48:46Z (1 hour ago)
 ```
 
-N.B. All of this string is necessary to activate a licence - not just the
-signature.
+Don't worry if you don't see all of these fields - some of them are only
+visible once you have activated a license. 
+
+The UUID in the `ID` field is unique to your cluster and is the only information 
+you need in order to obtain your first license.
+
+## Ondat - Obtaining a Personal Licence
+
+A personal license is aimed at individuals looking to try Ondat themselves before
+running it any business-critical situation. We're happy to provide guidance and 
+best-effort support with the Ondat community through our [slack channel](https://storageos.slack.com/) 
+or directly by email via info@ondat.io.
+
+To enable self-service for licenses, we provide a simple API. To request a license with curl, run:
+
+```shell
+curl -XPOST --data '{
+"clusterID": "704dd165-9580-4da4-a554-0acb96d328cb",
+"name": "Sally Forth",
+"country": "GB",
+"company": "Acme Ltd",
+"companySize": 100,
+"email": "sally@acme.io"
+}' https://t.api.ondat.io/license
+```
+
+```
+{
+  "code": "200",
+  "message": "License processed, please check your email for a download link."
+}
+```
+
+> Be sure to change the details to reflect your own!
+
+Do note that:
+- `clusterID` must be correct for your cluster, or the license won't apply!
+- `company` and `companySize` are totally optional, feel free to omit them
+- `country` can be a country in ISO 3166 format or the English name of the country
+- `email` will be validated and must be available and non-disposable
+
+> If you receive an `Internal Service Error`, double-check that all the information you've 
+submitted fits the above requirements!
+
+Now check your inbox! You will have received an automatically generated email with a link to
+your license file. If you don't see it, be sure to give it a few minutes and check your spam folder!
+
+The `Download license` button on the email will work as many times as you'd like to retrieve the license - 
```

Comment:
__Download license__ should be formatted in bold as that is how all screen elements are formatted. This `formatting` is only used for code elements

Reply:
Fixed! Thanks.